srcpkgs/python3-frozendict/template

```diff
@@ -1,19 +1,25 @@
 # Template file for 'python3-frozendict'
 pkgname=python3-frozendict
-version=2.0.7
+version=2.1.0
 revision=1
 wrksrc="frozendict-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3"
 checkdepends="python3-pytest"
 short_desc="Immutable mapping for Python3"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Joel Beckmeyer <joel@beckmeyer.us>"
 license="LGPL-3.0-only"
 homepage="https://github.com/Marco-Sulla/python-frozendict"
 distfiles="${PYPI_SITE}/f/frozendict/frozendict-${version}.tar.gz"
-checksum=a68f609d1af67da80b45519fdcfca2d60249c0a8c96e68279c1b6ddd92128204
+checksum=0189168749ddea8601afd648146c502533f93ae33840eb76cd71f694742623cd
 
 do_check() {
-	PYTHONPATH="$(cd build/lib* && pwd)" pytest3 -k 'not test_empty_sub'
+	PYTHONPATH="$(cd build/lib* && pwd)" pytest3 \
+	 --deselect=test/test_coold.py::test_repr \
+	 --deselect=test/test_coold.py::test_str \
+	 --deselect=test/test_coold.py::test_format \
+	 --deselect=test/test_frozendict_c.py::test_repr \
+	 --deselect=test/test_frozendict_c.py::test_str \
+	 --deselect=test/test_frozendict_c.py::test_format
```

Comment:
Could you add a comment here about why the tests are being skipped?

Reply:
Sure thing, done.